samples/outlook/01-compose-basics/get-item-subject.yaml

```diff
@@ -0,0 +1,48 @@
+id: outlook-01-compose-basics-get-item-subject
+name: Get Item Subject
+description: Gets the subject of a compose item
+host: OUTLOOK
+api_set: {}
+script:
+    content: |
+        $("#run").click(run);
+
```

Comment:
I would remove the extra newline, and only leave one.  Ditto for the next samples.